### ISSUE TITLE
Adding Spring Boot Web Services fails current example

### ISSUE THREAD

**Issue by franciszabala** [awslabs/aws-serverless-java-container#256]
* Framework version: 2.1.1.RELEASE or 1.5.20.RELEASE
* Implementations: Spring Boot

## Scenario
After adding this maven dependency:        

```
     	<dependency>
      		<groupId>org.springframework.boot</groupId>
      		<artifactId>spring-boot-starter-web-services</artifactId>
    	</dependency>

```

executing `mvn clean package` keeps throwing off:

`org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is java.lang.UnsupportedOperationException: Only dispatcherServlet is supported`

## Expected behavior
Should run as it would normally.

## Actual behavior
Stops at that error.

First it "dispatcherServlet" is passed as the first parameter in the function
`ServletRegistration.Dynamic addServlet` and after that, "messageDispatcher" pass causing the if statement ` if ("dispatcherServlet".equals(s)) ` to fail

## Steps to reproduce
Just add the maven dependency above

## Others
I need to create a Soap client and based on sample codes I gathered, that dependency is needed.

One work around is I copied SpringBootLambdaContainerHandler.java to my source folder, modified it to just return null. It does work as let me proceed with testing but it might have side effects.

```
} else {
                //throw new UnsupportedOperationException("Only dispatcherServlet is supported");
            	return null;
}
```

**Comment by sapessi** [comment]
Hi @franciszabala, this should be fixed for release 1.4 - I have a test implementation in the core branch of this repo. You can use the instructions from my last comment in #239 to test yourself.

**Comment by sapessi** [comment]
Release 1.4 is out on maven central. The [release notes](https://github.com/awslabs/aws-serverless-java-container/releases/tag/aws-serverless-java-container-1.4) include a list of all the changes. Closing this issue.
